src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2002,6 +2002,26 @@ internal static string ConcatenateStringPathArguments(CommandElementAst stringAs
                 {
                 }
             }
+
+            var argumentCompletionsAttribute = parameter.CompiledAttributes.OfType<ArgumentCompletionsAttribute>().FirstOrDefault();
+            if (argumentCompletionsAttribute != null)
+            {
+                try
```

Comment:
You are calling internal code which doesn't throw, so the try/catch is unnecessary.

Reply:
Fixed.